rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -473,6 +473,7 @@ namespace vk
 		case VK_IMAGE_LAYOUT_UNDEFINED:
 		case VK_IMAGE_LAYOUT_PREINITIALIZED:
 			fmt::throw_exception("Attempted to transition to an invalid layout");
+		default: break;
```

Comment:
Move default to the group with UNDEFINED and PREINITIALIZED. This should not just fall through.